web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,21 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-to-show">
+                    <div class="view_all_stream_link_container">
```

Comment:
I think a better class name here would be `subscribed_streams_tab_empty_text` and similarly below as well something like `all_streams_tab_empty_text`